vm/devices/storage/disk_ramdisk/src/lib.rs

```diff
@@ -1,20 +1,24 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-//! RAM-backed disk backend implementation.
+//! RAM-backed disk layer implementation.
 
 #![forbid(unsafe_code)]
 #![warn(missing_docs)]
 
 pub mod resolver;
 
 use anyhow::Context;
+use disk_backend::layered::DiskLayer;
+use disk_backend::layered::LayerConfiguration;
+use disk_backend::layered::LayerIo;
+use disk_backend::layered::LayeredDisk;
+use disk_backend::layered::SectorMarker;
+use disk_backend::layered::UnmapBehavior;
+use disk_backend::layered::WriteNoOverwrite;
 use disk_backend::zerodisk::InvalidGeometry;
```

Comment:
this might be a good PR to fix this weird zerodisk dependency in ramdisk. not major, but a weird dep to see

Reply:
I removed zerodisk since we don't use it for anything anymore.

Reply:
I mean, it serves as a simple "null disk" example implementation, but sure, we can just add it back in the future if we ever need it...